crates/bevy_render/src/mesh/mesh.rs

```diff
@@ -237,11 +237,10 @@ impl Mesh {
     /// The direction the vertex normal is facing in.
     /// Use in conjunction with [`Mesh::set_attribute`]
     pub const ATTRIBUTE_NORMAL: &'static str = "Vertex_Normal";
-    /// The direction of the vertex tangent. Used for normal mapping
-    pub const ATTRIBUTE_TANGENT: &'static str = "Vertex_Tangent";
-
     /// Where the vertex is located in space. Use in conjunction with [`Mesh::set_attribute`]
     pub const ATTRIBUTE_POSITION: &'static str = "Vertex_Position";
+    /// The direction of the vertex tangent. Used for normal mapping
+    pub const ATTRIBUTE_TANGENT: &'static str = "Vertex_Tangent";
```

Comment:
This should stay next to normal.

Reply:
How do I signal to cargo fmt not to immediately make this change again?

Reply:
`#[rustfmt::ignore]` maybe?

Reply:
Nope, crashes with error: invalid attribute.

Looks like `#[rustfmt::skip]` is the correct incantation.

Reply:
Which, of course, doesn't actually stop it from reordering. @bjorn3, before I waste an hour on this, how much do you care?

Reply:
I personally think we could just as easily disable the option - 7212b70478c8465b178611794d5738703eb9f6f1 doesn't explain why it was enabled.

Additionally, it currently seems like the option needs work, see https://github.com/rust-lang/rustfmt/issues/2792#issuecomment-397917400

Reply:
I agree with you there. @cart, do we have your blessing?

Reply:
Yup lets just disable it. Using order to semantically group things does seem valid in some contexts.

Reply:
This option has been removed!